app/javascript/call_tool/CallToolView.js

```diff
@@ -231,12 +169,8 @@ class CallToolView extends Component {
     });
   }
 
-  instructionsMessageId() {
-    if (this.props.restrictedCountryCode) {
-      return 'call_tool.instructions_without_country';
-    } else {
-      return 'call_tool.instructions';
-    }
+  jjkj() {
```

Comment:
After a closer look,  was worried that this wasn't caught as an error but I just realised that the function I've mistakenly renamed isn't used anymore.